internal/controllers/topology/cluster/conditions_test.go

```diff
@@ -418,13 +432,15 @@ func TestReconcileTopologyReconciledCondition(t *testing.T) {
 				UpgradeTracker: func() *scope.UpgradeTracker {
 					ut := scope.NewUpgradeTracker()
 					ut.ControlPlane.PendingUpgrade = false
+					ut.MachineDeployments.MarkRollingOut("md0-abc123")
 					ut.MachineDeployments.MarkPendingUpgrade("md1-abc123")
 					return ut
 				}(),
 				HookResponseTracker: scope.NewHookResponseTracker(),
 			},
-			wantConditionStatus: corev1.ConditionFalse,
-			wantConditionReason: clusterv1.TopologyReconciledMachineDeploymentsUpgradePendingReason,
+			wantConditionStatus:  corev1.ConditionFalse,
+			wantConditionReason:  clusterv1.TopologyReconciledMachineDeploymentsUpgradePendingReason,
+			wantConditionMessage: "MachineDeployment(s) md1-abc123 upgrade to version v1.22.0 on hold. MachineDeployment(s) md0-abc123 are rolling out",
 		},
 		{
 			name:         "should set the condition to false is some machine deployments have not picked the new version because other machine deployments are rolling out (unavailable replica)",
```

Comment:
```suggestion
			name:         "should set the condition to false if some machine deployments have not picked the new version because other machine deployments are rolling out (unavailable replica)",
```